### PR TITLE
Use the basic Forms application type instead of the Mvx one

### PR DESCRIPTION
**Pull request by martijn00**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)

Bug fix

### :arrow_heading_down: What is the current behavior?

You need to extend the Mvx type. Forms can't handle that one correctly

### :new: What is the new behavior (if this is a feature change)?

Use the basic Forms type. People could still use the mvx type if they want.

### :boom: Does this PR introduce a breaking change?

Yes

### :bug: Recommendations for testing

Run the playground

### :memo: Links to relevant issues/docs
Fixes https://github.com/MvvmCross/MvvmCross/issues/2512
Fixes: https://github.com/MvvmCross/MvvmCross/issues/2223

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [x] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide))
- [x] Rebased onto current develop

### CODE REVIEW

**Comment by nmilcoff** [comment]
Shouldn't we just remove our application class?

**Comment by nickrandolph** [comment]
@martijn00 @nmilcoff we should see whether this issue (ie Onstart not being called when using MvxApplication) has been fixed in the latest version of XF that just came out. If it has, I still think we accept this PR but I would leave the MvxApplication just in case devs want to use it

**Comment by martijn00** [comment]
@Cheesebaron It is all done, but Forms is not working, but neither is it in the develop branch, so i'm not sure what to do here. I'm trying to fix the playground some time soon.

**Comment by nickrandolph** [comment]
Hey guys apologies I’ve been a bit absent. I’ll try to jump in tomorrow and see if I can work out the issue with XF.

File: Projects/Playground/Playground.Forms.Uwp/Setup.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
```
Comment (nickrandolph) [inline Projects/Playground/Playground.Forms.Uwp/Setup.cs]: @martijn00 this file shouldn't be included in the UWP project any more

**Comment by nickrandolph** [comment]
@martijn00 can you rebase and remove the UWP setup class and then this should be good to go.

**Comment by martijn00** [comment]
@nickrandolph done!

**Comment by nmilcoff** [comment]
Sorry guys but I still don't get why are keeping our own FormsApplication class if it's buggy? 🤔

**Comment by martijn00** [comment]
It's actually not buggy, it works as expected. I think it might have been a bug in Xamarin.Forms. Our version has some advantages, so we could keep it, but i think we want to use the native type so people can choose themselves.

**Pull request merged**